app/src/main/java/org/oppia/android/app/home/promotedlist/PromotedStoryListViewModel.kt

```diff
@@ -49,6 +50,25 @@ class PromotedStoryListViewModel(
     }
   }
 
+  fun getRecentlyPlayedActivityTitle(): RecentlyPlayedActivityTitle {
+    with(promotedActivityList.promotedStoryList) {
+      return when {
+        suggestedStoryList.isNotEmpty() -> {
+          if (recentlyPlayedStoryList.isEmpty() && olderPlayedStoryList.isEmpty()) {
+            RecentlyPlayedActivityTitle.RECOMMENDED_STORIES
+          } else
+            RecentlyPlayedActivityTitle.STORIES_FOR_YOU
```

Comment:
```suggestion
          } else RecentlyPlayedActivityTitle.STORIES_FOR_YOU
```

Per https://developer.android.com/kotlin/style-guide#braces braces are always needed for if-else statements unless the expression can fit on one line.

Reply:
Done